subsys/bluetooth/controller/ticker/ticker.c

```diff
@@ -654,15 +654,18 @@ static u8_t ticker_resolve_collision(struct ticker_node *nodes,
 		while (id_head != TICKER_NULL) {
 			struct ticker_node *ticker_next = &nodes[id_head];
 
+			/* Accumulate ticks_to_expire for each node */
+			acc_ticks_to_expire += ticker_next->ticks_to_expire;
+			if (acc_ticks_to_expire > ticker->ticks_slot) {
+				break;
```

Comment:
Am I right in saying that this is an optimization - breaking here? (instead of possibly traversing more tickers)